crmsh/ra.py

```diff
@@ -49,7 +49,7 @@ def crm_resource(opts):
 
 @utils.memoize
 def can_use_lrmadmin():
```

Comment:
I doubt the codes related to lrmadmin already deprecated
Let me confirm that
If yes, then we can skip this issue

Reply:
Another solution https://github.com/readthedocs/readthedocs-sphinx-ext/pull/111/files and https://github.com/readthedocs/readthedocs-sphinx-ext/issues/106